public/locales/en/translation.json

```diff
@@ -28,8 +28,11 @@
     "Cannot determine vaccination status": "Cannot determine vaccination status",
     "Issuer recognized": "Issuer recognized",
     "Issuer not recognized": "Issuer not recognized",
-    "Only valid SMART Health Card QR Codes are currently supported.": "Only valid <0>SMART\u00AE Health Card</0> QR Codes<br/>are currently supported.",
+    "Only valid SMART Health Card QR codes are currently supported.": "Only valid <0>SMART\u00AE Health Card</0> QR codes<br/>are currently supported.",
     "This SMART Health Card cannot be verified.": "This <0>SMART\u00AE Health Card</0> cannot be<br/>verified. It may have been corrupted.",
+    "Unable to verify issuer.": "Unable to verify <0>SMART\u00AE Health Card</0> issuer. Please check internet access and try again later.",
+    "Only valid SMART Health Card QR codes are currently supported. Please contact your issuer for assistance.": "Only valid <0>SMART\u00AE Health Card</0> QR codes are currently supported. Please contact your issuer for assistance.",
```

Comment:
Would it be helpful to be a little more specific and say "Please contact the issuer of your SMART Health Card for assistance?"

Reply:
Looks like @nkarip made the update in https://github.com/dvci/shc-web-verifier/pull/56/commits/177b2c9d56c3642558e68b3c6aa2c0263e14b686. Thanks!